doc/source/installation.rst

```diff
@@ -9,20 +9,15 @@ Quick start
 
    export PETSC_CONFIGURE_OPTIONS="--download-suitesparse"
    curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/master/scripts/firedrake-install
-   python3 firedrake-install
+   python3 firedrake-install --install icepack
 
 * Active the Firedrake virtual environment::
 
    source <path/to/firedrake>/bin/activate
 
-* Clone and install icepack::
-
-   git clone https://github.com/icepack/icepack
-   cd icepack
-   pip3 install -e .
-
 * Run the icepack unit tests to make sure everything's working::
 
+   cd firedrake/src/icepack
```

Comment:
```suggestion
   cd <path/to/firedrake>/src/icepack
```

will make the instructions more consistent with what I have above about activating the virtual environment.

Reply:
I just pushed a slightly different fix. If one uses `$VIRTUAL_ENV` then the user can just paste the code.